test/jdk/java/lang/Thread/virtual/stress/Skynet.java

```diff
@@ -32,7 +32,7 @@
  * @test id=Z
  * @requires vm.debug == true & vm.continuations
  * @requires vm.gc.Z
- * @run main/othervm/timeout=300 -XX:+UnlockDiagnosticVMOptions
+ * @run main/othervm/timeout=400 -XX:+UnlockDiagnosticVMOptions
```

Comment:
Increasing the timeout is fine. There isn't anything in the bug report to suggest anything else.